tests/expectations/bitflags.cpp

```diff
@@ -27,11 +27,11 @@ struct AlignFlags {
     return *this;
   }
 };
-static const AlignFlags AlignFlags_AUTO = (AlignFlags){ .bits = 0 };
-static const AlignFlags AlignFlags_NORMAL = (AlignFlags){ .bits = 1 };
-static const AlignFlags AlignFlags_START = (AlignFlags){ .bits = 1 << 1 };
-static const AlignFlags AlignFlags_END = (AlignFlags){ .bits = 1 << 2 };
-static const AlignFlags AlignFlags_FLEX_START = (AlignFlags){ .bits = 1 << 3 };
```

Comment:
Why not keeping the `(StructName)` in C++?

Reply:
It occur error on MSVC. MSVC said it is non-standard.
> struct_literal.cpp(14): error C4576: a parenthesized type followed by an initializer list is a non-standard explicit type conversion syntax

Reply:
Does doing `AlignFlags { ... }` in C++ and `(AlignFlags) { .. }` in C work?

Reply:
`AlignFlags{ ... }` is working on MSVC! I add new commit doing this.